src/main/java/org/gnuhpc/bigdata/service/UserService.java

```diff
@@ -0,0 +1,111 @@
+package org.gnuhpc.bigdata.service;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+import org.gnuhpc.bigdata.config.WebSecurityConfig;
+import org.gnuhpc.bigdata.constant.GeneralResponseState;
+import org.gnuhpc.bigdata.model.GeneralResponse;
+import org.gnuhpc.bigdata.model.User;
+import org.gnuhpc.bigdata.utils.CommonUtils;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.*;
+
+@Getter
+@Setter
+@Log4j
+@Service
+public class UserService {
+  private HashMap<Object, Object> accounts;
+
+  public List<String> listUser() {
+    List<String> userList = new ArrayList<>();
+    try {
+      accounts = CommonUtils.yamlParse(WebSecurityConfig.SECURITY_FILE_PATH);
+      Iterator iterator = accounts.entrySet().iterator();
+      while (iterator.hasNext()) {
+        Map.Entry entry = (Map.Entry) iterator.next();
```

Comment:
Please modify this loop to java 9 stream coding style

Reply:
I will fix it asap.